components/service/glean/docs/baseline.md

```diff
@@ -1,10 +1,14 @@
 # The `baseline` ping
+
+## Description
 This ping is intended to provide metrics that are managed by the library itself, and not explicitly
 set by the application or included in the application's `metrics.yaml` file.
 
-The `baseline` ping is automatically sent when the application is moved to the *background* and it includes
+## Scheduling
+The `baseline` ping is automatically sent when the application is moved to the [background](pings.md#defining-background-state) and it includes
 the following fields:
 
+## Contents
 | Field name | Type | Description |
 |---|---|---|
 | `duration` | Timespan | The duration, in seconds, of the last foreground session |
```

Comment:
Since you're fixing this, any chance you could also fix [bug 1525578](https://bugzilla.mozilla.org/show_bug.cgi?id=1525578)? Metrics are not exactly at the top level, so additional indication on where to find them might be relevant for this or the general `metrics.md` file.